third_party/tonic/dart_persistent_value.cc

```diff
@@ -40,6 +40,7 @@ void DartPersistentValue::Clear() {
 
   auto dart_state = dart_state_.lock();
   if (!dart_state) {
+    value_ = nullptr;
```

Comment:
Add a comment that we are leaking the value because we are trying to delete it after dart_state_ has been deleted.  That's what's happening here, no?